microprofile/grpc/client/src/test/java/io/helidon/microprofile/grpc/client/GrpcClientBuilderTest.java

```diff
@@ -80,8 +78,6 @@ public void assertServiceOne(GrpcClientBuilder builder) {
         assertThat(grpcDescriptor, is(notNullValue()));
         assertThat(grpcDescriptor.getFullMethodName(), is("ServiceOne/foo/unary"));
         assertThat(grpcDescriptor.getType(), is(io.grpc.MethodDescriptor.MethodType.UNARY));
-        assertThat(grpcDescriptor.getRequestMarshaller(), is(instanceOf(JavaMarshaller.class)));
-        assertThat(grpcDescriptor.getResponseMarshaller(), is(instanceOf(JavaMarshaller.class)));
```

Comment:
Shouldn't we still check for "stub" marshaller, just to make sure the service has been configured correctly?